nexus/db-lookup/src/lookup.rs

```diff
@@ -26,8 +26,10 @@ use nexus_types::identity::Resource;
 use omicron_common::api::external::Error;
 use omicron_common::api::external::InternalContext;
 use omicron_common::api::external::{LookupResult, LookupType, ResourceType};
+use omicron_uuid_kinds::AccessTokenKind;
 use omicron_uuid_kinds::AlertReceiverUuid;
 use omicron_uuid_kinds::AlertUuid;
+use omicron_uuid_kinds::ConsoleSessionKind;
```

Comment:
in this file, we actually do need to reference the kind types directly (in the `LookupResource` macros) so referring to the expanded form of the type alias elsewhere is fine with me.